tests/scripts/thread-cert/node.py

```diff
@@ -1572,6 +1572,71 @@ def remove_service(self, enterpriseNumber, serviceData):
         self.send_command(cmd)
         self._expect_done()
 
+    def get_child_table(self) -> Dict[int, Dict[str, Any]]:
+        """Get the table of attached children."""
+        cmd = 'child table'
+        self.send_command(cmd)
+        output = self._expect_command_output(cmd)
+
+        #
+        # Example output:
+        # | ID  | RLOC16 | Timeout    | Age        | LQ In | C_VN |R|D|N|Ver|CSL|QMsgCnt| Extended MAC     |
+        # +-----+--------+------------+------------+-------+------+-+-+-+---+---+-------+------------------+
+        # |   1 | 0xc801 |        240 |         24 |     3 |  131 |1|0|0|  3| 0 |     0 | 4ecede68435358ac |
+        # |   2 | 0xc802 |        240 |          2 |     3 |  131 |0|0|0|  3| 1 |     0 | a672a601d2ce37d8 |
+        # Done
+        #
+
+        headers = self.__split_table_row(output[0])
+
+        table = {}
+        for line in output[2:]:
+            line = line.strip()
+            if not line:
+                continue
+
+            fields = self.__split_table_row(line)
+            col = lambda colname: self.__get_table_col(colname, headers, fields)
+
+            id = int(col("ID"))
+            r, d, n = int(col("R")), int(col("D")), int(col("N"))
+            mode = f'{"r" if r else ""}{"d" if d else ""}{"n" if n else ""}'
+
+            child = {
+                'id': int(id),
+                'rloc16': int(col('RLOC16'), 16),
+                'timeout': int(col('Timeout')),
+                'age': int(col('Age')),
+                'lq_in': int(col('LQ In')),
+                'c_vn': int(col('C_VN')),
+                'mode': mode,
+                'extaddr': col('Extended MAC')
+            }
+
+            if 'Ver' in headers:
+                child['ver'] = int(col('Ver'))
+
+            if 'CSL' in headers:
+                child['csl'] = bool(int(col('CSL')))
+
+            if 'QMsgCnt' in headers:
+                child['qmsgcnt'] = int(col('QMsgCnt'))
```

Comment:
Just curious, why we have the `if` checks here? Is it to check if `Ver` or `CSL` or `QMsgCnt` are included or not? I think they are always included in the table, right?

Reply:
> Just curious, why we have the if checks here? Is it to check if Ver or CSL or QMsgCnt are included or not?

Yes. 

>  I think they are always included in the table, right?

Yes. I copied the code from `otci.py`, which is compatible with older OT versions.  
Should be fined to remove here.

Reply:
removed